net: lib: download_client: remove APN support (#5983)
nrfconnect/sdk-nrf: applications/serial_lte_modem/doc/FOTA_AT_commands.rst

  It indicates to the modem the credential of the security tag used for establishing a secure connection for downloading the image.
  It is associated with the certificate or PSK.
  Specifying the ``<sec_tag>`` is mandatory when using HTTPS.
* The ``<pdn_id>`` parameter is an integer.

>   It represents the Packet Data Network ID (PDN ID), other than the default PDN, to use for downloading.

Reviewer: ```suggestion
  It represents the Packet Data Network (PDN) ID that can be used instead of the default PDN for downloading.
```
